frontend/src/components/parties/RelationshipProfile.js

```diff
@@ -0,0 +1,155 @@
+import { fetchMineRecordById } from "@/actionCreators/mineActionCreator";
+import { getPartyRelationshipTypesList, getPartyRelationships } from "@/selectors/partiesSelectors";
+import { getMines } from "@/selectors/mineSelectors";
+import Loading from "@/components/common/Loading";
+import * as router from "@/constants/routes";
+import CustomPropTypes from "@/customPropTypes";
+import * as String from "@/constants/strings";
+
+/**
+ * @class RelationshipProfile - profile view for party relationship types
+ */
+
+const TabPane = Tabs.TabPane;
+
+const propTypes = {
+  fetchMineRecordById: PropTypes.func.isRequired,
+  fetchPartyRelationshipTypes: PropTypes.func.isRequired,
+  fetchPartyRelationships: PropTypes.func.isRequired,
+  partyRelationships: PropTypes.arrayOf(CustomPropTypes.partyRelationship),
+  partyRelationshipTypes: PropTypes.arrayOf(CustomPropTypes.dropdownListItem),
+  match: PropTypes.object,
+  mines: PropTypes.arrayOf(CustomPropTypes.mine).isRequired,
+};
+
+const defaultProps = {
+  partyRelationships: [],
+  partyRelationshipTypes: [],
+};
+
+export class RelationshipProfile extends Component {
+  componentDidMount() {
+    const { id, typeCode } = this.props.match.params;
+    this.props.fetchPartyRelationshipTypes();
+    this.props.fetchPartyRelationships({ types: typeCode });
+
+    const mine = this.props.mines[id];
+    if (!mine) {
+      this.props.fetchMineRecordById(id);
+    }
+  }
+
+  getPartyRelationshipTypeLabel(partyRelationship) {
+    const partyRelationshipType = this.props.partyRelationshipTypes.find(
+      ({ value }) => value === partyRelationship.mine_party_appt_type_code
+    );
+    return (partyRelationshipType && partyRelationshipType.label) || String.EMPTY;
+  }
+
+  render() {
+    const { id } = this.props.match.params;
+    const mine = this.props.mines[id];
+
+    const isLoaded =
+      this.props.partyRelationshipTypes.length > 0 &&
+      this.props.partyRelationships.length > 0 &&
+      mine;
+
+    if (isLoaded) {
+      return (
+        <div className="profile">
+          <div className="profile__header">
+            <div className="inline-flex between">
+              <h1 className="bold">
+                {this.getPartyRelationshipTypeLabel(this.props.partyRelationships[0])} History
+              </h1>
+            </div>
+            <div className="inline-flex between">
+              <div className="inline-flex">
+                <p>
+                  <Link to={router.MINE_SUMMARY.dynamicRoute(mine.guid, "contact-information")}>
+                    {mine && mine.mine_name}
+                  </Link>
+                </p>
+              </div>
+            </div>
+          </div>
+          <div className="profile__content">
+            <Tabs activeKey="history" size="large" animated={{ inkBar: true, tabPane: false }}>
+              <TabPane tab="History" key="history">
+                <div>
+                  <Row type="flex" style={{ textAlign: "center" }}>
```

Comment:
Not necessary to do for this PR as there is a ticket to address this, 

But every data table going forward should be the ant design table, not rows/cols.